js/performance.ts

```diff
@@ -24,6 +20,6 @@ export class Performance {
     assert(msg.Any.NowRes === baseRes.innerType());
     const res = new msg.NowRes();
     assert(baseRes.inner(res) != null);
-    return res.time().toFloat64() - this.timeOrigin;
+    return res.seconds().toFloat64() * 1e9 + res.subsecNanos();
```

Comment:
I thought now returns millis?

Reply:
I think this needs to be float of millis

Reply:
switched from seconds to millis

Reply:
After few thoughts it was working, see: https://doc.rust-lang.org/std/time/struct.Duration.html#examples-6

I've to revert the changes so.

Reply:
Just this line needed to change

Reply:
I've seen i've misunderstood what you said and fixed everything.